src/StarCluster/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+if (AMReX_SPACEDIM EQUAL 3)
+    # Define a custom target that runs the Python script to produce the input perturbations file
+    add_custom_command(
+        OUTPUT ${CMAKE_SOURCE_DIR}/tests
+        COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/perturbation.py --kmin=2 --kmax=64 --size=128 --alpha=2 --f_solenoidal=1.0
+        COMMAND cp -p ${CMAKE_CURRENT_SOURCE_DIR}/zdrv.hdf5 ${CMAKE_SOURCE_DIR}/tests
+        DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/perturbation.py --kmin=2 --kmax=64 --size=128 --alpha=2 --f_solenoidal=1.0
+        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
+    )
+
```

Comment:
Would this work

Reply:
I don't think we want to run this at build time. Can you change it so it runs during the test instead?

Reply:
@BenWibking how do I do that? I think you mentioned using a bash script or something? Can you give an example

Reply:
The easiest way might be to create a separate test to run `perturbation.py`, and then make it a dependency of the `StarCluster` test:
```
add_test(NAME ComputeStarClusterPerturbations COMMAND perturbation.py --kmin=2 --kmax=64 --size=128 --alpha=2 --f_solenoidal=1.0 WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/tests)
add_test(NAME StarCluster COMMAND star_cluster StarCluster.in WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/tests)
set_tests_properties(ComputeStarClusterPerturbations PROPERTIES FIXTURES_SETUP    StarCluster_fixture)
set_tests_properties(StarCluster PROPERTIES FIXTURES_REQUIRED StarCluster_fixture)
```

Reply:
`COMMAND perturbation.py` doesn't work; see my modification.

The StarCluster test seems to _hang_ . I will try to dig into it if I get time.